aiohttp/web_request.py

```diff
@@ -409,7 +409,8 @@ def post(self):
                     out.add(field.name, ff)
                 else:
                     value = yield from field.read(decode=True)
-                    if content_type.startswith('text/'):
+                    if content_type is None or \
+                            content_type.startswith('text/'):
```

Comment:
FYI

https://tools.ietf.org/html/rfc7578#page-5

and also these chapters

> 5.1.2.  Interpreting Forms and Creating multipart/form-data Data
> 
>    Some applications of this specification will supply a character
>    encoding to be used for interpretation of the multipart/form-data
>    body.  In particular, HTML 5 [W3C.REC-html5-20141028] uses
> 
>    o  the content of a "_charset_" field, if there is one;
> 
>    o  the value of an accept-charset attribute of the <form> element, if
>       there is one;
> 
>    o  the character encoding of the document containing the form, if it
>       is US-ASCII compatible;
> 
>    o  otherwise, UTF-8.

> 5.1.3.  Parsing and Interpreting Form Data
> 
>    While this specification provides guidance for the creation of
>    multipart/form-data, parsers and interpreters should be aware of the
>    variety of implementations.  File systems differ as to whether and
>    how they normalize Unicode names, for example.  The matching of form
>    elements to form-data parts may rely on a fuzzier match.  In
>    particular, some multipart/form-data generators might have followed
>    the previous advice of [RFC2388] and used the "encoded-word" method
>    of encoding non-ASCII values, as described in [RFC2047]:
> 
>       encoded-word = "=?" charset "?" encoding "?" encoded-text "?="
> 
>    Others have been known to follow [RFC2231], to send unencoded UTF-8,
>    or even to send strings encoded in the form-charset.
> 
>    For this reason, interpreting multipart/form-data (even from
>    conforming generators) may require knowing the charset used in form
>    encoding in cases where the _charset_ field value or a charset
>    parameter of a "text/plain" Content-Type header field is not
>    supplied.
>